searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -242,14 +242,17 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
             return reply;
         }
 
-        MatchParams params(searchContext.getDocIdLimit(), _rankSetup->getHeapSize(), _rankSetup->getArraySize(),
+        const Properties & rankProperties = request.propertiesMap.rankProperties();
+        uint32_t heapSize = search::fef::indexproperties::hitcollector:: HeapSize::lookup(rankProperties,
+                                                                                          _rankSetup->getHeapSize());
```

Comment:
There is an extra space.
Also it is better to add a using HeapSize = search::fef::indexproperties::hitcollector::HeapSize.
Having to break long lines due to very long names just makes a mess.